### DISCUSSION TITLE
Orchard core for Web development

### DISCUSSION THREAD

**Discussion by FireEagle97** [OrchardCMS/OrchardCore#15088] [Q&A]
Hi everyone, I'm planning to work as a development freelancer and I was looking for a framework that can help me build websites that can be scalable and easy to maintain and match my knowledge in C#. I found OrchardCMS pretty awesome as it is free and open source and works on .Net framework which I'm familiar with. I did the dojo3 course and finished it with many questions: 
-Does anyone here have experience with building websites with Orchardcore? How did you find it?
-Can anyone send me a GitHub repo of an example and see how the project structure works?
- How to modify the theme and how can I see the theme files if for example switch from theme to theme but its folder is not in the project folder?
- How can I see the files of the Admin theme?
Thank you for your help in advance.

**Comment by rwxzig** [comment]
[Here ](https://github.com/OrchardCMS/OrchardCore/tree/main/src/OrchardCore.Themes)you can find the themes that comes with Orchard.
[This](https://github.com/OrchardSkills) user has also some interesting repositories, but they are from outdated versions, updating them should not be a big problem.
And of course [Lombiq](https://github.com/Lombiq) is a go to place when it comes to Orchard.

**Comment by Piedone** [comment]
Thanks for the shoutout for Lombiq @cptzig! In addition to what was mentioned, since you mention the [Dojo Course3 tutorial series](https://orcharddojo.net/orchard-training/dojo-course-3-the-full-orchard-core-tutorial), also check out these:

- [Orchard Core news on Orchard Dojo](https://orcharddojo.net/)
- [Show Orchard](https://showorchard.com/)
- [Lombiq's Open-Source Orchard Core Extensions](https://github.com/Lombiq/Open-Source-Orchard-Core-Extensions)
